pypeit/coadd2d.py

```diff
@@ -1312,7 +1295,7 @@ def __init__(self, spec2d_files, spectrograph, par, det=1, offsets=None, weights
             self.objid_bri, self.slitidx_bri, self.spatid_bri, self.snr_bar_bri = \
                 np.repeat(user_objid, self.nexp), _slitidx_bri, user_slit, None
         elif len(self.stack_dict['specobjs_list']) > 0 and (offsets == 'auto' or weights == 'auto'):
-            self.objid_bri, self.slitidx_bri, self.spatid_bri, self.snr_bar_bri = \
+            self.objid_bri, self.spat_pixpos_bri, self.slitidx_bri, self.spatid_bri, self.snr_bar_bri = \
```

Comment:
`self.spat_pixpos_bri` should be defined also in the `if` above and the `else` below. Even if they are  just`None`